live-examples/html-examples/content-sectioning/css/header.css

```diff
@@ -0,0 +1,32 @@
+@import url('https://fonts.googleapis.com/css?family=Dancing+Script');
+
+header.pageHeader {
+  background-color: rgb(200,245,190);
+  border-bottom: 2px solid rgb(90, 150, 90);
+  margin: 0;
+  height: 120px;
+}
+
+header.pageHeader > img {
+  height: 120px;
+  float: left;
+  margin-right: 20px;
+}
+
+header.pageHeader > h1 {
+  font: bold 42px "Dancing Script", cursive;
```

Comment:
Wee seem to discourage people from using pixels for font-size: https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#Pixels

Reply:
Okay, this has been informative. I hadn't realized just how much we were trying to make all the things ideal here, rather than focusing on just making the example show the element in question. I will work to make these more adherent to all the things going forward.